<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>-3</y>
      <width>1231</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="KioskHome">
     <widget class="QLabel" name="MessageLabel">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>330</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="EducationButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>469</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Education</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LogisticsButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>187</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Logistics</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HealthcareButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>184</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Healthcare</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BankingButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>466</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Banking</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="LogisticsHome">
     <widget class="QPushButton" name="DeliveryButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>391</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delivery</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TakeawayButton">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>230</y>
        <width>391</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Takeaway</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="MobileNumberHome">
     <widget class="QPushButton" name="MobileCancelButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>140</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MobileConfirmButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>100</y>
        <width>140</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>-10</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Enter the Mobile Number</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push1Button">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>220</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push2Button">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>220</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push3Button">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>220</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push6Button">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>330</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push5Button">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>330</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push4Button">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>330</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push9Button">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>440</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push8Button">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>440</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push7Button">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>440</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserInputMobileLabel">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushClearButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>550</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push0Button">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>550</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushDeleteButton">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>550</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="OtpHome">
     <widget class="QPushButton" name="OtpCancelButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>140</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>-10</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter the OTP Number</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OtpConfirmButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>100</y>
        <width>140</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserInputOtpLabel">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="push3Button_3">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>220</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push6Button_3">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>330</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push2Button_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>220</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push7Button_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>440</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push8Button_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>440</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push4Button_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>330</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push1Button_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>220</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push5Button_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>330</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push0Button_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>550</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushClearButton_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>550</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push9Button_3">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>440</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushDeleteButton_3">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>550</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="ProcessingMessageLabel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>731</width>
        <height>431</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>PROCESSING</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="RcvLogoLabel">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>-20</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="VamanitKioskResources/vamanitkioskresources.qrc">:/RcvLogo/rcvLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="rcvNameLogo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>660</y>
      <width>1361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="pixmap">
     <pixmap resource="VamanitKioskResources/vamanitkioskresources.qrc">:/idrbtfooter/idrbtfooter.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="idrbtcalogoLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="VamanitKioskResources/vamanitkioskresources.qrc">:/idrbtcalogo/idrbtcalogo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="VamanitKioskResources/vamanitkioskresources.qrc"/>
 </resources>
 <connections/>
</ui>
